ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -188,15 +188,18 @@ struct MSFListCellView: View {
                 }
             }, label: {
                 HStack(spacing: 0) {
-                    let hasTitle = !state.title.isEmpty
-                    let labelAccessoryInterspace = tokens.labelAccessoryInterspace
-                    let labelAccessorySize = tokens.labelAccessorySize
-                    let sublabelAccessorySize = tokens.sublabelAccessorySize
+                    let hasTitle: Bool = !state.title.isEmpty
+                    let labelAccessoryInterspace: CGFloat = tokens.labelAccessoryInterspace
+                    let labelAccessorySize: CGFloat = tokens.labelAccessorySize
+                    let sublabelAccessorySize: CGFloat = tokens.sublabelAccessorySize
 
                     if let leadingView = state.leadingView {
-                        leadingView
-                            .frame(width: tokens.leadingViewSize, height: tokens.leadingViewSize)
-                            .padding(.trailing, tokens.iconInterspace)
+                        HStack(alignment: .center, spacing: 0) {
+                            leadingView
+                                .frame(width: tokens.leadingViewSize, height: tokens.leadingViewSize)
+                        }
+                        .frame(width: tokens.leadingViewArea)
```

Comment:
Why do we need the leadingViewArea token?
Shouldn't the overall width be just `leadingViewSize + (2 * horizontalCellPadding)`?

Reply:
Not necessarily - the leadingViewSize has varying sizes, but the leadingViewArea stays the same at 48px constantly. That formula would cause the area to have varying sizes